polyaxon_client/tracking/build.py

```diff
@@ -70,13 +75,14 @@ def create(self,
 
         return self
 
+    @check_offline
     def _update(self, patch_dict):
-        self.client.build_job.update_build(username=self.username,
-                                           project_name=self.project_name,
-                                           job_id=self.job_id,
-                                           patch_dict=patch_dict,
-                                           background=True)
+        if self.client:
```

Comment:
Do we need this change here, the decorator should take care of that.

Reply:
No, we don't. Fixed.